apps/dashboard/src/components/Students/StatsContent.tsx

```diff
@@ -64,15 +64,21 @@ const StatsContent: React.FC<{
       },
       {
         label: intl("stats.student.time.total"),
-        value: Duration.from(data.minutes.total.toString()).format("ar"),
+        value: Duration.from(data.minutes.total.toString()).format({
+          language: "ar",
+        }),
       },
       {
         label: intl("stats.student.time.ratified"),
-        value: Duration.from(data.minutes.ratified.toString()).format("ar"),
+        value: Duration.from(data.minutes.ratified.toString()).format({
+          language: "ar",
+        }),
       },
       {
         label: intl("stats.student.time.canceled"),
-        value: Duration.from(data.minutes.canceled.toString()).format("ar"),
+        value: Duration.from(data.minutes.canceled.toString()).format({
+          language: "ar",
+        }),
```

Comment:
This is now how it should be done. Don't update the `Duration` class. Just write a local function to achive this here. Don't forget to revert the changes done in the `Duration` class.